packages/scheduled-tasks/src/lib/strategies/ScheduledTaskRedisStrategy.ts

```diff
@@ -0,0 +1,69 @@
+import { container } from '@sapphire/framework';
+import Bull, { Job, JobOptions, Queue, QueueOptions } from 'bull';
+import type { ScheduledTaskCreateRepeatedTask, ScheduledTasksTaskOptions } from '../types';
+import type { ScheduledTaskBaseStrategy } from '../types/ScheduledTaskBaseStrategy';
+
+export interface ScheduledTaskRedisStrategyOptions {
+	queue?: string;
+	bull?: QueueOptions;
+}
+
+export interface ScheduledTaskRedisStrategyJob {
+	task: string;
+	payload?: unknown;
+}
+
+export class ScheduledTaskRedisStrategy implements ScheduledTaskBaseStrategy {
+	public readonly options: QueueOptions;
+	public readonly queue: string;
+
+	private client!: Queue;
```

Comment:
```suggestion
	private bullClient!: Queue;
```

Just to avoid any confusion between this and the DJS Client.